configs/mainnet.json

```diff
@@ -218,5 +218,13 @@
       "perpTradingBoost": 3
     }
   },
-  "cross_selling_source_tokens": ["swth"]
+  "cross_selling_source_tokens": ["swth"],
+  "typeform_widget_config": {
+    "surveyLink": "https://switcheonetwork.typeform.com/to/C1rheplK",
+    "endTime": "2024-07-26T23:00:00Z",
+    "pages": [
+      "/account/overview",
+      "/account/balance"
+    ]
+  }
```

Comment:
this only allows one survey to be ongoing at one time, it should be a map/array of survey params instead so that we can support multiple survey forms